### ISSUE TITLE
QuicClient.CreateStream is hardcoded to 1.

### ISSUE THREAD

**Issue by angedelamort** [Vect0rZ/Quic.NET#3]
I've changed my local version to:

`public QuicStreamContext CreateStream(ulong streamId = 1, StreamType streamType = StreamType.ClientBidirectional);`

in order to keep the compatibility.

**Comment by Vect0rZ** [comment]
Good catch. We can use a NumberSpace class instance here, it's part of QuicNet.Infrastructure. We  can substitute the streamId optional parameter with a private NumberSpace in the QuicClient, and leave the streamType in place, so a client can choose whether is Uni/Bidirectional.

Still, let's merge that one in, 

Thanks!
